src/Bucket/Bucket.jl

```diff
@@ -587,21 +592,14 @@ function make_update_aux(model::BucketModel{FT}) where {FT}
         )
         ρ_sfc = surface_air_density(p, model.atmos)
 
-        snow_cover_fraction = heaviside.(Y.bucket.σS)
-        q_sat =
-            saturation_specific_humidity.(
-                p.bucket.T_sfc,
-                Y.bucket.σS,
-                ρ_sfc,
-                Ref(model.parameters),
-            )
         # This relies on the surface specific humidity being computed
         # entirely over snow or over soil. i.e. the snow cover fraction must be a heaviside
         # here, otherwise we would need two values of q_sfc!
-        p.bucket.q_sfc = @. (
-            (FT(1.0) - snow_cover_fraction) *
-            β(Y.bucket.W, model.parameters.W_f) *
-            q_sat + snow_cover_fraction * q_sat
+        @. p.bucket.q_sfc = saturation_specific_humidity(
```

Comment:
just FYI this function "saturation_specific_humidity" returns the value over ice if snow covered and over liq if not.